api4/file_test.go

```diff
@@ -287,7 +285,7 @@ func testUploadFiles(
 	}
 
 	for i, open := range openers {
-		f, cl, err = open()
+		f, _, err := open()
```

Comment:
Whole method looks pretty suspicious. I just change some parts. I would like to remove https://github.com/mattermost/mattermost-server/blob/01a6309268cb9bbd9f51ed4362170b827dccbfec/api4/file_test.go#L220 as well, as its not used anywhere.

Reply:
Go for it.

Reply:
This one is old, will be deprecated soon. It *is* suspicious, I just re-worked the code, tests are all in api4. I would recommend leaving it to be deprecated with the old code altogether

Reply:
Would you accept my changes? I want add a test to CI to detect these kind of code flaws. Because of this I want to fix this soon.